doc/hacking.rst

```diff
@@ -98,6 +98,19 @@ For integration tests, we use a system similar to `Cram tests
 
 .. _ppx_expect:      https://github.com/janestreet/ppx_expect
 
+When running dune inside tests, the ``INSIDE_DUNE`` environment variable is set.
+This has the following effects:
+
+* Change the default root detection behaviour to use the current directory
+  rather than the top most ``dune-project`` / ``dune-workspace`` file.
+* Be less verbose various times Dune outputs a user message.
```

Comment:
```suggestion
* Be less verbose in the various times Dune outputs a user message.
```
…or “during the various times” ??
Feels like there’s something missing between “verbose” and “various”

Reply:
Hmm, sounds ok when I say it out loud, but I am no expert on grammar.

Reply:
```suggestion
* Be less verbose when Dune outputs a user message.
```
tada this is better.

Reply:
Yes! Perfect!